tests/test_muonclip_attention.py

```
@@ -0,0 +1,203 @@
+"""Tests for attention instrumentation helpers."""
+
+import torch
+import torch.nn as nn
+
+import torch
```

Ruff (0.14.4)

Redefinition of unused `torch` from line 3

Remove definition: `torch`

(F811)